src/components/Settings/People/DeleteConfirmationDialog.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-
+import { TextField } from '../../UI/FormItems';
```

Comment:
Use the `TextField` component from material UI.
`import {TextField} from from '@material-ui/core';`